app/src/main/java/org/oppia/android/app/player/state/itemviewmodel/ImageRegionSelectionInteractionViewModel.kt

```diff
@@ -41,8 +43,8 @@ class ImageRegionSelectionInteractionViewModel private constructor(
     val schemaObject = interaction.customizationArgsMap["imageAndRegions"]
     schemaObject?.customSchemaValue?.imageWithRegions?.imagePath ?: ""
   }
-
   val isAnswerAvailable = ObservableField<Boolean>(false)
+  val lastSelectedRegion = ObservableField<LabeledRegion>(rawUserAnswer.imageRegionSelection)
```

Comment:
Why does this need to be public?

Reply:
Because it is used in `image_region_selection_interaction_item.xml` file for lastSelectedRegion attribute

Reply:
Ah, I think I saw that in a previous change and forgot about it. :) Thanks for the clarification.